### PR TITLE
Add transformer entry point

### PR DESCRIPTION
**Pull request by christophe-rannou**
#96

### CODE REVIEW

**Comment by XciD** [comment]
UP

File: pmml-evaluator/src/main/java/org/jpmml/evaluator/TransformerContext.java [hunk 1]
```diff
@@ -66,6 +61,11 @@ private FieldValue declareMissing(FieldName name) {
         return declare(name, (FieldValue) null);
     }
 
+    @Override
+    protected FieldValue prepare(FieldName name, Object value) {
+        throw new NotImplementedException();
```
Comment (vruusmann) [inline pmml-evaluator/src/main/java/org/jpmml/evaluator/TransformerContext.java]: Should be `java.lang.UnsupportedOperationException` instead.

The `org.jpmml.evaluator.NotImplementedException` has a different meaning - "the PMML specification defines some feature/nehaviour, but this is not yet implemented in the JPMML-Evaluator library"

**Comment by vruusmann** [comment]
What's the use case for 64-bit integer values? Counting milliseconds since 01-01-1970?

Please comment here: https://github.com/jpmml/jpmml-evaluator/issues/141

**Comment by vruusmann** [comment]
Duplicate of #96

**Pull request closed without merging**